alyx/actions/models.py

```diff
@@ -155,6 +156,22 @@ def save(self, *args, **kwargs):
         return super(Surgery, self).save(*args, **kwargs)
 
 
+class Project(BaseModel):
+    name = models.CharField(
+        max_length=255, unique=True, blank=True, help_text="Project name")
+
+    description = models.CharField(
+        max_length=1023, blank=True, help_text="Description of the project")
+
+    repositories = models.ManyToManyField(
```

Comment:
Not sure why we need repositories to be associated with Project. Is it for information only, or will this be used to decide where to save files?

Reply:
If I remember correctly, we thought it could be used to say whether a particular project is associated with IBL and should be uploaded, or not.